libraries/botbuilder-core/src/testAdapter.ts

```diff
@@ -327,36 +437,34 @@ export class TestAdapter extends BotAdapter implements ExtendedUserTokenProvider
         if (!context || !context.activity) {
             throw new Error('testAdapter.getTokenStatus(): context with activity is required');
         }
-        
+
         if (!userId && (!context.activity.from || !context.activity.from.id)) {
             throw new Error(`testAdapter.getTokenStatus(): missing userId, from or from.id`);
         }
 
         const filter = (includeFilter ? includeFilter.split(',') : undefined);
-        if(!userId) {
+        if (!userId) {
             userId = context.activity.from.id;
         }
 
-        const match = this._userTokens.filter(x => x.ChannelId === context.activity.channelId 
-                                            && x.UserId === userId 
-                                            && (!filter || filter.includes(x.ConnectionName)));
+        const match = this._userTokens.filter(x => x.channelId === context.activity.channelId
+            && x.userId === userId
+            && (!filter || filter.includes(x.connectionName)));
 
-        if (match && match.length > 0)
-        {
+        if (match && match.length > 0) {
```

Comment:
`Array.prototype.filter` won't return undefined. Also, consider combining this code into a filter/map.

```typescript
return this._userTokens.filter(...).map(token => ({
  ConnectionName: token.connectionName,
  HasToken: true,
  ServiceProviderDisplayName: token.connectionName,
}));
```